src/auth/anonymous.strategy.ts

```diff
@@ -0,0 +1,16 @@
+import { Strategy } from 'passport-anonymous'
```

Comment:
@apolkingg8 This is not needed yet, but it's nice if we want to allow a subset of features to be available publicly. Eventually, it may make sense to add authorization with roles, so we can specify endpoint access with a decorator: `@Roles([Role.Anonymous, Role.Admin])`